galaxy_importer/loaders/content.py

```diff
@@ -21,14 +21,20 @@
 from pathlib import Path
 import re
 import shutil
-from subprocess import Popen, PIPE
+from subprocess import Popen, PIPE, TimeoutExpired
 import yaml
+import json
+from jsonschema import validate, ValidationError, SchemaError
```

Comment:
ansible-lint is getting schema checking support for patterns so we really don't need it here.

RE: https://github.com/ansible/ansible-lint/pull/4653 and https://github.com/ansible/ansible-lint/pull/4673